pkg/restapi/issuer/operation/operations.go

```diff
@@ -176,6 +176,16 @@ func (c *Operation) createCredential(subject []byte) ([]byte, error) {
 
 	httpClient := http.DefaultClient
 
+	cred, err := sendHTTPRequest(req, httpClient, http.StatusCreated)
+	if err != nil {
+		return nil, err
+	}
+
+	req, err = http.NewRequest("POST", c.vcsURL+"/store", bytes.NewBuffer(cred))
+	if err != nil {
+		return nil, err
+	}
+
 	return sendHTTPRequest(req, httpClient, http.StatusCreated)
```

Comment:
you sending create cred twice https://github.com/trustbloc/edge-sandbox/pull/54/files#diff-1b8efd39a1789cdada8e6d9350d7f050R179